utils/key.go

```diff
@@ -0,0 +1,32 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package utils
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"testing"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+// Key contains an ecdsa private key field as well as an address field
+// obtained from converting the ecdsa public key.
+type Key struct {
+	Address    common.Address
+	PrivateKey *ecdsa.PrivateKey
+}
+
+// NewKey generates a new key pair and returns a pointer to a [Key].
+func NewKey(t *testing.T) *Key {
```

Comment:
nit: I think this belongs to testutils etc. Importing testify and testing pkgs here seems a bit weird.

Reply:
Moved from utils to internal/testutils in #1552